model/licensing.py

```diff
@@ -1306,6 +1312,10 @@ class DeliveryMechanism(Base, HasFullTableCache):
     _cache = HasFullTableCache.RESET
     _id_cache = HasFullTableCache.RESET
 
+    __table_args__ = (
+        UniqueConstraint('content_type', 'drm_scheme'),
+    )
+
```

Comment:
In this case a new UniqueConstraint is created to replace a unique index, and then a second unique partial index is created.